.github/workflows/ci.yml

```
@@ -478,7 +478,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version-file: .nvmrc
+          node-version: 20
```

Let's add a comment here:

```suggestion
          # Run integration tests with the oldest supported node version.
          node-version: 20
```